### DISCUSSION TITLE
Automatically play next book in series

### DISCUSSION THREAD

**Discussion by DamienZaid** [advplyr/audiobookshelf#2035] [Ideas]
I have lots of audiobooks for my kids that are part of different series. At the moment I always have to manually select the next one, once a book is finished.

It would be great if the next book in a series could automatically play, once the previous book is finished.

I'm talking specifically about the Android app

**Comment by nichwall** [comment]
You should already be able to do this with a playlist (since you said "part of different series").

Otherwise I can see why auto playing the next book in a series would be nice, but I personally don't want the next book to happen since I download my books at home and don't want it streaming the next book on data (so maybe configurable?)

**Reply by DamienZaid** [reply]
I only said "part of different series" because I have multiple series, but I'd only like to play the next one in the series of the book that is currently playing

**Reply by nichwall** [reply]
Ah, thanks for clarifying. I wasn't sure if you were wanting to automatically play books from different series

**Reply by DarkAngelGH** [reply]
On the Android app, this still doesn't work for Collections.  When one book finishes, the player stops, rather than going to the next one.  It would indeed be convenient if there were some kind of "Play All" button for series, and that the player would simply start the next book when the previous one was finished, but I'd be happy if it simply worked with collections.  There doesn't seem to be much point to a collection if playback stops after every book, forcing manual intervention to continue.

**Comment by DamienZaid** [comment]
A "play series" option in the context menu could be an option to allow playing the next book automatically and if you don't want that, you could use the regular "play".
